### ISSUE TITLE
Lost comunication to usb device 

### ISSUE THREAD

**Issue by chinemys** [exploitagency/github-rfpwnon#1]
after some time sending data, I get this error "Lost communication to usb device ..waiting 3 seconds, then retrying ...
latest firmware rfcat yard stick one compiled with sdcc

**Comment by exploitagency** [comment]
Does it end up retrying?
Use some sort of spectrum analyzer/watch the waterfall using another device like a hackrf or rtlsdr.

**Comment by chinemys** [comment]
yes, try again, and come back up a few times and then crash ... I'm using sdr, I see the communication until then it stops as written above. I tried several USB sockets and several notebook s, always the same problem
error:
Transmitting...
Lost communication to USB device.. waiting 3 seconds, then retrying.
Traceback (most recent call last):
  File "rfpwnon.py", line 222, in <module>
    ConfigureD(d)
  File "rfpwnon.py", line 109, in ConfigureD
    d.setMdmModulation(MOD_ASK_OOK)
  File "/usr/local/lib/python2.7/dist-packages/rflib/chipcon_nic.py", line 589, in setMdmModulation
    self.getRadioConfig()
  File "/usr/local/lib/python2.7/dist-packages/rflib/chipcon_nic.py", line 385, in getRadioConfig
    bytedef = self.peek(0xdf00, 0x3e)
  File "/usr/local/lib/python2.7/dist-packages/rflib/chipcon_usb.py", line 838, in peek
    r, t = self.send(APP_SYSTEM, SYS_CMD_PEEK, struct.pack("<HH", bytecount, addr))
  File "/usr/local/lib/python2.7/dist-packages/rflib/chipcon_usb.py", line 700, in send
    return self.recv(app, cmd, wait)
  File "/usr/local/lib/python2.7/dist-packages/rflib/chipcon_usb.py", line 664, in recv
    raise(ChipconUsbTimeoutException())
rflib.chipcon_usb.ChipconUsbTimeoutException: Timeout waiting for USB response.

**Comment by zwt2135** [comment]
Same here I have this issue as well.
